example/timer/timer-example.service

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Inago Timer Example Service
+
+[Service]
+ExecStart=/bin/bash -c "echo Beep"
```

Comment:
we may want to add a `RemainAfterExit=yes` to this file, so it stays with `active/exited` in systemd.